db/data_migrations/20200108105400_create_backend_apis.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'progress_counter'
+
+class CreateBackendApis < ActiveRecord::DataMigration
+  def up
+    services = Service.where.has { not_exists(BackendApiConfig.except(:order).select(:id).by_service(BabySqueel[:services].id)) }
+
+    return puts 'Nothing to do.' if services.empty?
+
+    progress = ProgressCounter.new(services.count)
+
+    services.includes(:backend_api_configs).find_each(batch_size: 100) do |service|
+      progress.call do
+        next if service.backend_api_configs.any?
+
+        service_name = service.name
+        backend_api = service.account.backend_apis.create(
+          system_name: service.system_name,
+          name: "#{service_name} Backend",
+          description: "Backend of #{service_name}",
+          private_endpoint: service.proxy&.[]('api_backend')
```

Comment:
Maybe the reason why the "bug" I reported in https://github.com/3scale/porta/pull/1517/files#r364296373 is not causing bigger problems is because of the removal `api_backend` also in 2.7. In the end, only services whose proxy has an `api_backend`, i.e. old services created before 2.7, will get a corresponding Backend created.